CONTRIBUTING.md

```diff
@@ -2,6 +2,15 @@
 
 ## Development
 
+1. Register a new [application](https://developers.google.com/cast/docs/registration).
```

Comment:
Could you expand on this step? eg note that its a "Custom Receiver", and does the name matter? does it have to match a property in the source or can it be anything?

Reply:
Those are some good points 👍